.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,28 @@
+---
+name: integration_tests
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - main
```

Comment:
Q: `gofail` has only `master` branch, is `main` added as a future thought in case of rename?

Reply:
Yes, this is my intention, as I see all other etcd repos are migrating from master to main.